content/influxdb/cloud-serverless/reference/cli/influx/v1/auth/create.md

```diff
@@ -15,7 +15,7 @@ updated_in: CLI v2.0.3
 #### Not supported in InfluxDB Cloud
 **InfluxDB Cloud** does not support InfluxDB 1.x compatible authorizations.
```

Comment:
I assume this either doesn't work or isn't advisable for Serverless, but I don't actually know.